lib/src/Uncertainty/Distribution/Normal.cxx

```diff
@@ -704,7 +704,9 @@ Point Normal::getSkewness() const
 /* Get the standard representative in the parametric family, associated with the standard moments */
 Distribution Normal::getStandardRepresentative() const
 {
-  return Normal(0.0, 1.0);
+  Normal standard(getDimension());
```

Comment:
actually its never called in n-d in the transformations, but doesnt hurt

Reply:
Ok, so we agree. I guess that this was just a comment on your side. So you agree on this comment can be resolved?